debezium-core/src/main/java/io/debezium/pipeline/spi/SnapshotResult.java

```diff
@@ -33,7 +41,8 @@ public OffsetContext getOffset() {
 
     public static enum SnapshotResultStatus {
         COMPLETED,
-        ABORTED;
+        ABORTED,
+        SKIPPED
```

Comment:
Does this need a semicolon after `SKIPPED`? The list used to have one; I hadn't noticed that before.

Reply:
The only time a semicolon is mandatory is when an enumeration defines constants and methods, otherwise its technically unnecessary; otherwise I believe its merely a personal style preference. 

FWIW, it does not look like I see an option in the format plugin to control this.  There may be a way to handle this with checkstyle though I haven't checked yet.